components/datetime/src/skeleton/mod.rs

```diff
@@ -379,25 +374,6 @@ mod test {
         };
     }
 
-    /// Skeletons are represented in bincode as a vec of field, but bincode shouldn't be completely
-    /// trusted, test that the bincode gets validated correctly.
-    struct TestInvalidSkeleton(Vec<Field>);
```

Comment:
This was dead code?

Reply:
Yes, I'm not sure why this wasn't detected before...